files/orchestrate/src/consume/consume.ts

```diff
@@ -17,7 +18,7 @@ export const consume = async () => {
       console.error('Transaction failed with error: ', value.errors)
     } else {
       console.log('RequestId:', value.id)
-      console.log('Receipt: ', value.receipt)
+      console.log('Receipt: ', util.inspect(value.receipt, false, null, true))
```

Comment:
I'd advise using `JSON.stringify(value.receipt, null, 2)` here to pretty print the receipt as a JSON object w/ an indent level of 2. `util.inspect` can truncate sometimes.

Reply:
Thanks, updated